test/gator/test/test.bats

```diff
@@ -29,7 +29,7 @@ match_yaml_msg () {
   want_msg="${2}"
   violation_index="${3:-0}"
 
-  if ! got=$(echo -n "$yaml_output" | yq eval ".[${violation_index}].msg" - --exit-status); then
+  if ! got=$(echo -n "$yaml_output" | yq eval ".[${violation_index}].result.msg" - --exit-status); then
```

Comment:
notably, if you are using `gator test ... -o yaml` and string processing/ feeding that into something else, this may break you. 

I know `gator` is technically in "alpha" but I still want to call this out to the reviewers. The old gator would look something like 

```
- ${type.result}
```

now this looks like

```
- result: ${type.Result}
  violatingObject: ${unstructred.Unstructured}
  trace: ${trace}
```

That's inline w how we do the `json` outoput

Reply:
We could add a note in the release note for this.